documentation/specs/proposed/RAR_caching_onepager.md

```diff
@@ -0,0 +1,54 @@
+## RAR caching
```

Comment:
Please explain the abbreviation the first time it's used so it's clear to the readers.